doc/build/orm/mapped_sql_expr.rst

```diff
@@ -34,12 +34,13 @@ will provide for us the ``fullname``, which is the string concatenation of the t
 Above, the ``fullname`` attribute is interpreted at both the instance and
 class level, so that it is available from an instance::
 
-    some_user = session.query(User).first()
+    some_user = session.scalars(select(User)).first()
     print(some_user.fullname)
 
 as well as usable within queries::
 
-    some_user = session.query(User).filter(User.fullname == "John Smith").first()
+    some_user = session.scalars(select(User).where(User.fullname == "John Smith")).first()
```

Comment:
**mike bayer** (zzzeek) wrote:

.limit(1)

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3729